### DISCUSSION TITLE
软件包里mosdns 没有luci-app-mosdns和对应的中文包，能加入不？

### DISCUSSION THREAD

**Discussion by shaw21003** [immortalwrt/immortalwrt#1119] [General]
软件包里mosdns 没有luci-app-mosdns和对应的中文包，能加入不？

**Comment by science2468** [comment]
> 软件包里mosdns 没有luci-app-mosdns和对应的中文包，能加入不？

https://github.com/sbwml/luci-app-mosdns/releases/tag/v5.3.1

**Comment by m0eak** [comment]
同样需要，这样编译的时候不用再额外加一个feeds了
